op-node/node/node.go

```diff
@@ -373,7 +374,14 @@ func (n *OpNode) initL2(ctx context.Context, cfg *Config, snapshotLog log.Logger
 	if cfg.ConductorEnabled {
 		sequencerConductor = NewConductorClient(cfg, n.log, n.metrics)
 	}
-	n.l2Driver = driver.NewDriver(&cfg.Driver, &cfg.Rollup, n.l2Source, n.l1Source, n.beacon, n, n, n.log, snapshotLog, n.metrics, cfg.ConfigPersistence, &cfg.Sync, sequencerConductor)
+	// plasma DA client will be inactive unless enabled in the cli config.
+	storage := plasma.NewDAClient(cfg.Plasma.DAServerURL)
```

Comment:
Ideally we would not have to construct the storage layer outside of the PlasmaDA object.